docs/en-us/development/code-conduct.md

```diff
@@ -0,0 +1,68 @@
+## Code of Conduct
+
+The following Code of Conduct is based on full compliance with the [Apache Software Foundation Code of Conduct](https://www.apache.org/foundation/policies/conduct.html).
+
+### Development philosophy.
+ - **Consistent** code style, naming, and usage are consistent.  
+ - **Easy to read** code is ambiguous, easy to read and understand, when debugging one knows the intent of the code.

```

Comment:
the word ambiguous
differ from the original doc
maybe use clear nonambiguous obvious plain unambiguous unequivocal accurate instead